torch/onnx/symbolic_opset12.py

```diff
@@ -13,3 +13,7 @@
 def einsum(g, equation, tensor_list):
     tensors = sym_help._unpack_list(tensor_list)
     return g.op("Einsum", *tensors, equation_s=equation)
+
+@parse_args('s', 'v', 'v')
+def mse(g, reduction, input, target):
+    return g.op("MeanSquaredDistance", input, target, reduction=reduction)
```

Comment:
reduction is an int in aten::mse_loss but a string in ONNX. 
So we need to add some logic here to convert the int value to the corresponding string ("none", "mean", "sum")